scripts/execute_trajectory.py

```diff
@@ -0,0 +1,40 @@
+import sys
+import copy
+import rospy
+import moveit_commander
+import moveit_msgs.msg
+import geometry_msgs.msg
+from math import pi
+from std_msgs.msg import String
+from moveit_commander.conversions import pose_to_list
+import rosbag
+
+bag = rosbag.Bag('joint_states.bag')
+
+moveit_commander.roscpp_initialize(sys.argv)
+rospy.init_node('move_group_python_interface_tutorial', anonymous=True)
+
+group_name = "panda_arm"
+move_group = moveit_commander.MoveGroupCommander(group_name)
+
+joint_goal = move_group.get_current_joint_values()
+plan_msg = []
```

Comment:
`move_group.execute()` does not allow an argument of type `list`, but instead it accepts `RobotTrajectory` class.
so at the top,
```
from moveit_msgs.msg import RobotTrajectory, MultiDOFJointTrajectory, MultiDOFJointTrajectoryPoint
from trajectory_msgs.msg import JointTrajectory, JointTrajectoryPoint
```

```
joint_trajectory = JointTrajectory()
joint_trajectory.joint_names = [] # SPECIFY THE NAMES
joint_trajectory.joint_trajectory = JointTrajectoryPoint(positions=[]) # SPECIFY THE JOINT POSITIONS

plan = RobotTrajectory()
plan.joint_trajectory = joint_trajectory

move_group.execute(plan, wait=True)
```